Luqman011/PropertyPro-Lite
# Merge UI to gh-pages

**what does this PR do?**

merges UI to gh-pages

**how should this be manually tested?**
navigate to https://github.com/Luqman011/PropertyPro-Lite
then check the gh-pages branch
